### PR TITLE
changed to know where flow is used

### PR DESCRIPTION
**Pull request by edewit**
based on how many idps or clients it's used either
a popup or a modal dialog

fixes: #2509

### CODE REVIEW

**Comment by edewit** [comment]
less then eight clients or idps it has a popup: 
image
more then it it renders a dialog:
image

**Comment by edewit** [comment]
also changed the defaults names to the flow type:
image

File: src/authentication/components/UsedBy.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+type UsedByProps = {
+  authType: AuthenticationType;
+};
+
+const Label = ({ id, label }: { id: string; label: string }) => (
+  <>
+    <CheckCircleIcon
+      className="keycloak_authentication-section__usedby"
+      key={`icon-${id}`}
```
Comment (jonkoops) [inline src/authentication/components/UsedBy.tsx:27]: I don't think this `key` prop is needed as this is not inside of a loop.

File: src/authentication/components/UsedBy.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                ))}
+              </div>
+            }
+          >
+            <a href="javascript:;">
+              <Label id={id!} label={t(type!)} />
+            </a>
+          </Popover>
+        ) : (
+          <a href="javascript:;" onClick={toggle}>
```
[the remark this answers is not in the document]
Comment (edewit) [reply src/authentication/components/UsedBy.tsx:124]: But it adds a lot of padding and changes the font

[the remark this answers is not in the document]
Comment (edewit) [reply src/authentication/components/UsedBy.tsx:124]: image
removing all the padding:
image
but fixed it with a css now

**Pull request merged**